aeolus/global.h

```diff
@@ -17,17 +17,15 @@
     Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 */
 
-
 #ifndef __GLOBAL_H
 #define __GLOBAL_H
 
-
 #ifdef __APPLE__
 #include <machine/endian.h>
-#define __LITTLE_ENDIAN	__DARWIN_LITTLE_ENDIAN
-#define __BIG_ENDIAN	__DARWIN_BIG_ENDIAN
-#define __PDP_ENDIAN	__DARWIN_PDP_ENDIAN
-#define	__BYTE_ORDER	__DARWIN_BYTE_ORDER
+#define __LITTLE_ENDIAN __DARWIN_LITTLE_ENDIAN
+#define __BIG_ENDIAN    __DARWIN_BIG_ENDIAN
+#define __PDP_ENDIAN    __DARWIN_PDP_ENDIAN
+#define     __BYTE_ORDER      __DARWIN_BYTE_ORDER
```

Comment:
Very strange space optimisation :)

Reply:
What are we going to do about this ?

Reply:
It were TABs, now converted to 4 spaces, so a manual fix seems needed